client/src/java/com/zimbra/client/ZMailbox.java

```diff
@@ -2129,8 +2144,7 @@ public List<ZContact> getContactsForFolder(String folderid, String ids, ContactS
                 String[] temp = Arrays.copyOfRange(arr, start, end);
                 String subIds = String.join(",", temp);
 
-                Element req = getGetContactsRequest(folderid, sortBy, sync, attrs);
-                req.addAttribute(MailConstants.A_DEREF_CONTACT_GROUP_MEMBER, true);
+                Element req = getGetContactsRequest(folderid, sortBy, sync, attrs, true);
```

Comment:
since we have forceDerefGroupMember param available here why are we defaulting this request with true here?

Reply:
`req.addAttribute(MailConstants.A_DEREF_CONTACT_GROUP_MEMBER, true);` is always required when id is specified.
This is just moving the code in previous line 2133 to the new line 2183.